src/components/Accordion/Accordion.stories.tsx

```diff
@@ -1,29 +1,91 @@
-import React, { useState, FC } from 'react';
-import Accordion from './Accordion';
-import './_Accordion.scss'
-
-export default {
-  title: 'Accordion',
-  component: Accordion,
-};
-
-const accordionStyle: React.CSSProperties = {
-  marginRight: '5px',
-  marginTop: '5px',
-};
-
-  export const DefaultAccordion = (props) => {
-    return (
-      <React.Fragment>
-        <Accordion/>
-      </React.Fragment>
-    );
-  };
-
-export const DisabledAccordion = (props) => {
-  return (
-    <Accordion
-      disabled
-    />
-    )
-  };
+// created storybook to test the accordion component
+import React, {useState, useEffect} from 'react';
+import Accordion from '../Accordion';
+import { Icon } from '../../index';
+import './_Accordion.scss'
+
+
+
+export default {
+    title: 'Accordion',
+    component: Accordion,
+}
+
+export const DefaultAccordion = () => (
+    <div>
+        <p>A basic Accordion component.</p>
+    <Accordion
+      title={"Expansion Panel 1"}
+      content="This accordion is expanded by default"
+      expandIcon={<Icon name='angle down' size='mini' color='black' />}
+      collapseIcon={<Icon name='angle down' size='mini' color='black' />}
+      onChange={(expanded) => console.log(expanded)}  
+    />
+    <Accordion
+      title="Accordion 2"
+      content="This accordion is expanded by default"
+      expandIcon={<Icon name='angle down' size='mini' color='black' />}
+      collapseIcon={<Icon name='angle down' size='mini' color='black' />}
+      expanded={true}
+    />
+    <Accordion
+      title="Accordion 3"
+      content="This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks This is a test accordion content. Test test and test only. thanks"

```

Comment:
Maybe a good way to pull content from another component so it won't be too long.